samples/generated/express-embedded-auth-with-sdk/web-server/views/authenticator copy.mustache

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
```

Comment:
Do we need this file?

Reply:
Good catch! will regenerate the samples.